src/Akeneo/Connectivity/Connection/back/Domain/Webhook/Exception/ConnectionWebhookNotFoundException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Akeneo\Connectivity\Connection\Domain\Webhook\Exception;
+
+use Akeneo\Connectivity\Connection\Domain\DomainErrorInterface;
+use Symfony\Component\HttpFoundation\Response;
+use Throwable;
+
+/**
+ * @author    Willy Mesnage <willy.mesnage@akeneo.com>
+ * @copyright 2020 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ConnectionWebhookNotFoundException extends \DomainException implements DomainErrorInterface
+{
+    public function __construct(
+        $message = 'akeneo_connectivity.connection.webhook.error.not_found',
+        Throwable $previous = null
+    ) {
+        parent::__construct($message, Response::HTTP_NOT_FOUND, $previous);
```

Comment:
You can't use `Symfony\Component\HttpFoundation\Response` in our Domain
```suggestion
        parent::__construct($message, 0, $previous);
```